### ISSUE TITLE
Start Unity while unfocussed and pause it when it's reached a satisfactory start state

### ISSUE THREAD

**Issue by SethCram** [azesmway/react-native-unity#128]
Hi @azesmway thanks for maintaining this repo!! 

I recently upgraded from v0.3.1 to v1.0.10, and I noticed that previously, my Unity game would start loading by default even when not focused, requiring a manual pause on mount. In the new version, I'm looking for a way to manually enable this behavior. My Unity content is in a separate screen that loads in the background when a tab navigator is first initialized, but since the Unity tab isn't the first one focused, I'd like the game to load in the background so it's ready at scene 1 when first navigated to. However, I've tried using `unityRef.current.resumeUnity();` and `unityRef.current.windowFocusChanged(true);` when the tab is mounted, but neither approach has worked on Android. So far, the only way I've managed to start the game is by manually focusing the tab.

**Comment by IhsotasTon** [comment]
same here, confused me two days

**Comment by sajidhashmi** [comment]
any solution?

**Comment by SethCram** [comment]
@sajidhashmi nope, not that I know of. I'm considering just setting the initial route to the tab with the game in it, then navigating away so it atleast starts loading

**Comment by abhayrajput87** [comment]
hey, any solution?

**Comment by SethCram** [comment]
@abhayrajput87 I'm just setting the initial route to the tab with the game in it, then navigating away so it atleast starts loading.
